examples/next/pages/ui/components/storage/storage-manager/accept-all-files/index.page.tsx

```diff
@@ -0,0 +1,14 @@
+import { Amplify } from 'aws-amplify';
+import { StorageManager } from '@aws-amplify/ui-react-storage';
+import '@aws-amplify/ui-react/styles.css';
+import awsExports from './aws-exports';
+Amplify.configure(awsExports);
+
+export function StorageManagerExample() {
+  return (
+    <>
+      <StorageManager accessLevel="public" maxFileCount={1} showThumbnails />
```

Comment:
Thanks for adding this. It may be more valuable to test a scenario that has `acceptedFileTypes` with `*`. What do you think?

Reply:
+1

Reply:
updated to *